WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3564,27 +3566,25 @@ e_byzantium = {
 		
 		capital = 560 # Cherson
 		
-		hellenic_pagan = 600
-		bosporan_faith = 200
+		hellenic_pagan = 200
+		bosporan_faith = 600
+		tengri_pagan_reformed = 200
+		slavic_pagan_reformed = 50 # Crusade target weight
 		
 		allow = {
-			hidden_tooltip = {
-				OR = {
-					ai = no
-					AND = {
-						religion_group = christian
-						OR = { 
-							culture_group = byzantine
-							culture_group = latin
-						}
+			OR = {
+				religion = bosporan_faith
+				AND = {
+					religion_group = christian
+					OR = { 
+						culture_group = byzantine
+						culture_group = latin
+						has_romance_culture_trigger = yes
```

Comment:
I have already explained why ai = no stays, and I don't get why you added the Romance group.